plutus-core/plutus-core/src/PlutusCore/Parser/Builtin.hs

```diff
@@ -78,7 +80,11 @@ conBool =
 -- | Parser for lists.
 conList :: DefaultUni (Esc a) -> Parser [a]
 conList uniA = trailingWhitespace . inBrackets $
-    constantOf ExpectParensNo uniA `sepBy` symbol ","
+  constantOf ExpectParensNo uniA `sepBy` symbol ","
+
+-- | Parser for arrays.
+conArray :: DefaultUni (Esc a) -> Parser (Vector a)
+conArray uniA = Vector.fromList <$> conList uniA
```

Comment:
Are there any tests that should have caught this?  It doesn't matter because we'll have to add textual conformance tests later that will definitely catch this, but I had a vague idea that there were some property tests that involved parsing prettyprinted random ASTs.  I haven't tried to find those though.